okhttp/src/main/java/okhttp3/internal/ws/WebSocketReader.java

```diff
@@ -128,12 +132,25 @@ private void readHeader() throws IOException {
       throw new ProtocolException("Control frames must be final.");
     }
 
-    boolean reservedFlag1 = (b0 & B0_FLAG_RSV1) != 0;
+    boolean compressedFlag = (b0 & B0_FLAG_RSV1) != 0;
```

Comment:
I don't think we should rename this. The meaning of the flag only indicates compressed when the extension is specified. And the check below should still throw when the extension is not specified and this bit is set.

Reply:
Makes sense. Fixed.